packages/app-store/ee/routing-forms/components/FormActions.tsx

```diff
@@ -69,7 +69,7 @@ function NewFormDialog({ appUrl }: { appUrl: string }) {
       router.push(`${appUrl}/form-edit/${variables.id}`);
     },
     onError: () => {
-      showToast(`Something went wrong`, "error");
```

Comment:
We need to add the string to en/common.json as well so that it can be picked by crowdin and let's see if there are similar strings already there. We use shorter keys for strings e.g. this can be become something_went_wrong

Reply:
made the change

Reply:
There is already a translation key for the string. https://github.com/calcom/cal.com/blob/43a19126088e45e08ac79fcd6137cbcd20564ed6/apps/web/public/static/locales/en/common.json#L418